extensions/wikia/CommunityPage/models/CommunityPageDefaultCardsModel.class.php

```diff
@@ -15,7 +15,7 @@ public function getData( $modulesLimit ) {
 				'title' => wfMessage( 'communitypage-cards-create-page' )->text(),
 				'description' => wfMessage( 'communitypage-cards-create-page-description' )->text(),
 				'icon' => 'create-page',
-				'actionlink' => SpecialPage::getTitleFor( 'CreatePage' )->getLocalURL(),
+				'actionlink' => LinkHelper::forceLoginLink(SpecialPage::getTitleFor( 'CreatePage' ), $edit=false),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$edit" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)